README.md

```diff
@@ -3,91 +3,83 @@
+To install the latest version of PPM, run:
 
-#### Other branches
+```bash
+$ pip install ppafm
+```
 
-* **master_backup** - Old `master` branch was recently significantly updated and named `main`. For users who miss the old master branch, we provided a backup copy. However, this version is very old and its use is discouraged. If you miss some functionality or are not satisfied with the behavior of current `main` branch please let us know by creating an *issue*.
-* **PhotonMap** - implements the latest developments concerning sub-molecular scanning probe combined with Raman spectroscopy (TERS)y and fluorescent spectroscopy (LSTM).
-* **complex_tip** - Modification of probe-particle model with 2 particles allows a better fit to experimental results at the cost of additional fitting parameters.
+This should install the package and all its dependencies.
+Once the installation is completed, the following commands should be available:
 
-## Installation & running examples
+- `ppafm-generate-elff` - command-line interface to gelerate electrostatic force field.
+- `ppafm-generate-elff-point-charges` - command-line interface to gelerate electrostatic force field using point charges.
+- `ppafm-generate-ljff` - command-line interface to gelerate Lennard-Jones force field.
+- `ppafm-relaxed-scan` - command-line interface to run scan the sample with the probe particle.
+- `ppafm-plot-results` - command-line interface to plot the results of the simulation.
+- `ppafm-gui` - GUI application for interactive simulation of AFM images.
 
-All development and testing were done on **linux** OS (mostly ubuntu). However, there are also pip wheels for Windows available. See also the docker image below for a platform-independent way of running the code.
 
-#### Install & run CPU version
+### Install GPU GUI
 
-**Requirements:** Python3 (numpy,matplotlib) & C/C++ compiler (g++,make)
+To make sure the `ppafm-gui` command works without problems, you need to install QT5 library on your system.
+On Ubuntu, you can do this by running:
 
-##### First run: Graphene with point-charges
- 1. clone the repository: `clone https://github.com/Probe-Particle/ProbeParticleModel.git`
- 2. compile the C/C++ modules
-    * `cd ProbeParticleModel/Graphene`
-    * `make`
- 3. Navigate to examples directory `cd ProbeParticleModel/examples/Graphene`this example uses simple (Point-charges + Lennard-Jones)
- 4. run the example `./run.sh`
- 5. output directory `/examples/Graphene/Q-0.05K0.50/Amp2.0` should contain simulated images with tip charge -0.05e, stiffness 0.5N/m and ossicaltion amplitude 2.0A.
+```bash
+$ sudo apt install git python3-pip python3-pyqt5
```

Comment:
I think we can change this to just
```suggestion
$ sudo apt install python3-pyqt5
```
since you don't need git when installing from PyPI and you should have pip already if you managed to get here.

One thing to maybe note here is that this requires rights to install system packages, which you may not have. There is also the option of using Anaconda, which seems to manage qt5 support on it's own, and does not require administrative access. But it's an additional hurdle, so I would maybe make a wiki page with additional installation information like this.